pykern/pkcli/test.py

```diff
@@ -211,8 +212,18 @@ def _try(output, restartable):
             except Exception as e:
                 return _except(e, output, restartable)
             o = pkio.read_text(output)
-            if _COROUTINE_NEVER_AWAITED.search(o):
-                pkio.write_text(output, o + _FAILED_ON_WARNINGS)
+            if m := re.findall(_COROUTINE_NEVER_AWAITED, o):
+                pkio.write_text(
```

Comment:
The output is already written so append:

```py
with output.open(mode="a") as f:
    f.write("".join(f"ERROR: {x}\n" for x in m)
```

Extra newlines obscure other errors. At the moment, you are finding it hard to see this particular error. However, in your future, you'll be annoyed that this particular error is being highlighted, and the actual error is embedded somewhere else in the text.

There is no good solution to this particular problem. The types of errors we get are complex, especially because we are written concurrent code.